src/set_theory/game/nim.lean

```diff
@@ -303,73 +297,44 @@ lemma grundy_value_eq_mex_right : ∀ (G : pgame) [G.impartial],
   apply grundy_value_neg
 end
 
+/-- The Grundy value of the sum of two nim games with natural numbers of piles equals their bitwise
+xor. -/
 @[simp] lemma grundy_value_nim_add_nim (n m : ℕ) :
   grundy_value (nim.{u} n + nim.{u} m) = nat.lxor n m :=
```

Comment:
Just a remark: it's possible to generalize this theorem to all ordinals, with RHS `= ordinal.lxor n m` defined by applying `lxor` to each term in Cantor normal form (so maybe this operation could be called "natural xor"), cf. http://www.neverendingbooks.org/on2-conways-nim-arithmetics.

Reply:
Huh, I had no idea. Right now the Cantor Normal Form is very underdeveloped, but I'll write this down for a future PR.